.github/workflows/build-package-DevExpress.yml

```diff
@@ -0,0 +1,197 @@
+          Expand-Archive -Path $PackedFile -Destination $UnpackedDir -Force
+          $Params = @('sign',
+            '-kvt', '${{ secrets.AZURE_TENANT_ID }}',
+            '-kvu', '${{ secrets.CODE_SIGNING_KEYVAULT_URL }}',
+            '-kvi', '${{ secrets.CODE_SIGNING_CLIENT_ID }}',
+            '-kvs', '${{ secrets.CODE_SIGNING_CLIENT_SECRET }}',
+            '-kvc', '${{ secrets.CODE_SIGNING_CERTIFICATE_NAME }}',
+            '-tr', '${{ vars.CODE_SIGNING_TIMESTAMP_SERVER }}',
+            '-v')
+          Get-ChildItem "$UnpackedDir\lib" -Include @("*.dll") -Recurse | ForEach-Object {
+            AzureSignTool @Params $_.FullName
+          }
+          Remove-Item $PackedFile -ErrorAction SilentlyContinue | Out-Null
+          Compress-Archive -Path "$UnpackedDir\*" -Destination $PackedFile -CompressionLevel Optimal
+
+      - name: Upload nuget package
+        uses: actions/upload-artifact@v4.3.6
+        with:
+          name: package-nupkg
+          path: package/*.nupkg
+
+  publish:
+    name: Publish packages
+    runs-on: ubuntu-22.04
+    needs: [preflight, build-nuget]
+    environment: ${{ needs.preflight.outputs.package-env }}
+    if: ${{ fromJSON(needs.preflight.outputs.skip-publish) == false }}
+
+    steps:
+      - name: Download nuget package
+        uses: actions/download-artifact@v4
+        with:
+          name: package-nupkg
+          path: package
+
+      - name: Publish to nuget.org
+        shell: pwsh
+        run: |
+          $DryRun = [System.Boolean]::Parse('${{ needs.preflight.outputs.dry-run }}')
+          $NugetPackage = (Get-Item ./package/*.nupkg) | Resolve-Path -Relative
+
+          $PushArgs = @(
+            'nuget', 'push', "$NugetPackage",
+            '--api-key', '${{ secrets.NUGET_API_KEY }}',
+            '--source', 'https://api.nuget.org/v3/index.json',
+            '--skip-duplicate', '--no-symbols'
+          )
+          Write-Host "dotnet $($PushArgs -Join ' ')"
+          if ($DryRun) {
+            Write-Host "Dry Run: skipping nuget.org publishing!"
+          } else {
+            & 'dotnet' $PushArgs
+          }
+
+      - name: Create GitHub release
+        shell: pwsh
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        working-directory: package
+        run: |
+          $PackageVersion = '${{ needs.preflight.outputs.package-version }}'
+          $DryRun = [System.Boolean]::Parse('${{ needs.preflight.outputs.dry-run }}')
+
+          $HashPath = 'checksums'
+          $Files = Get-Item * | % { Get-FileHash -Algorithm SHA256 $_.FullName }
+          $Files | % { "$($_.Hash)  $(Split-Path $_.Path -Leaf)" } | Out-File -FilePath $HashPath -Append -Encoding ASCII
+
+          echo "::group::checksums"
+          Get-Content $HashPath
+          echo "::endgroup::"
+
+          $ReleaseTag = "v$PackageVersion"
+          $ReleaseTitle = "Devolutions Avalonia Themes v${PackageVersion}"
```

Comment:
I can't see this title appearing anywhere - should it be unique between the two projects/nugets?

Reply:
GitHub Releases doesn't support release channels, so we're releasing everything at once. This title is what shows up in GitHub Releases: https://github.com/Devolutions/avalonia-themes/releases. We're uploading the .nupkg files, but nobody really uses them directly, they're importing them from nuget.org